homeassistant/components/tuya/light.py

```diff
@@ -110,8 +110,9 @@ class TuyaLightEntityDescription(LightEntityDescription):
         ),
         # Not documented
         # Based on multiple reports: manufacturer customized Dimmer 2 switches
+        # Second Light is not SWITCH_LED_1, it is SWITCH_1
         TuyaLightEntityDescription(
-            key=DPCode.SWITCH_LED_1,
+            key=DPCode.SWITCH_1,
```

Comment:
This will break others, as this was added based on provided diagnostics.
It seems like you have another variant.

Reply:
I provided the original diagnostics. I was under the impression that it may have been some type of capy and paste error. Please let me know how I may be of assistance to resolve this correctly.

Reply:
Oh right, I see: <https://github.com/tuya/tuya-home-assistant/issues/781>
And <https://github.com/home-assistant/core/pull/65990/files>

Sorry about that!

Reply:
Thank you. Is there anything else that I need to do in order to correctly close out this PR and issue?

Reply:
Process the review comment below.

Reply:
Thank you